develop/using_utils.rst

```diff
@@ -43,10 +43,10 @@ operator. The format function uses the following string formatting characters.
 Here are a few examples to help elaborate on the above descriptions::
 
   >>> format("Error %q has been reported %n.  For more information, see %u.",
-             "AttributeError", (5, "time"), "http://supybot.com")
+             "AttributeError", (5, "time"), "https://limnoria.net")
 
   'Error "AttributeError" has been reported 5 times.  For more information,
-   see <http://supybot.com>.'
+   see <https://limnoria.net>.'
```

Comment:
I have no idea what these were originally supposed to be, but supybot.com didn't reply to me and maybe they were just examples and supybot.com didn't host developer documetnation directly?